docs/admin/iceberg-server.md

```diff
@@ -0,0 +1,1351 @@
+- An auxiliary service embedded in the Gravitino server, the CLASSPATH is `iceberg-rest-server/libs`.
+
+For detailed instructions on how to build and install the Gravitino server package,
+please refer to [the build guide](../develop/how-to-build.md) and [the installation guide](../install/install.md).
+To build the Gravitino Iceberg REST server package, use the command `./gradlew compileIcebergRESTServer -x test`.
+Alternatively, to create the corresponding compressed package in the distribution directory,
+use `./gradlew assembleIcebergRESTServer -x test`.
+The Gravitino Iceberg REST server package includes the following files:
+
+```text
+├─ ...
+└─ distribution/gravitino-iceberg-rest-server
+    ├─ bin/
+    │  └─ gravitino-iceberg-rest-server.sh    # Launching scripts.
+    ├─ conf/                                   # All configurations.
+    │  ├─ gravitino-iceberg-rest-server.conf  # Server configuration.
+    │  ├─ gravitino-env.sh                    # Environment variables, e.g. JAVA_HOME, GRAVITINO_HOME, etc.
+    │  ├─ log4j2.properties                   # log4j configurations.
+    │  └─ hdfs-site.xml & core-site.xml       # HDFS configuration files.
+    ├─ libs/                                   # Dependencies libraries.
+    └─ logs/                                   # Logs directory. Auto-created after the server starts.
+```
+
+## Server configuration
+
+There are distinct configuration files for standalone and auxiliary server:
+
+- `gravitino-iceberg-rest-server.conf` is used for the standalone server;
+- `gravitino.conf` is for the auxiliary server.
+
+Although the configuration files differ, the configuration items remain the same.
+
+Starting with version `0.6.0-incubating`, the prefix `gravitino.auxService.iceberg-rest.`
+for auxiliary server configurations has been deprecated.
+If both `gravitino.auxService.iceberg-rest.key` and `gravitino.iceberg-rest.key` are present,
+the latter will take precedence.
+The configurations listed below use the `gravitino.iceberg-rest.` prefix.
+
+### Configuration to enable Iceberg REST service in Gravitino server.
+
+<table>
+<thead>
+<tr>
+  <th>Configuration item</th>
+  <th>Description</th>
+  <th>Default value</th>
+  <th>Required</th>
+  <th>Since Version</th>
+</tr>
+</thead>
+<tbody>
+<tr>
+  <td><tt>gravitino.auxService.names</tt></td>
+  <td>
+    The auxiliary service name of the Gravitino Iceberg REST catalog service.
+    Use `iceberg-rest`.
+  </td>
+  <td>(none)</td>
+  <td>Yes</td>
+  <td>`0.2.0`</td>
+</tr>
+<tr>
+  <td><tt>gravitino.iceberg-rest.classpath</tt></td>
+  <td>
+    The CLASSPATH of the Gravitino Iceberg REST catalog service,
+    including the directory containing JARs and configuration.
+    It supports both absolute and relative paths.
+    For example, `iceberg-rest-server/libs,iceberg-rest-server/conf`.
+  </td>
+  <td>(none)</td>
+  <td>Yes</td>
+  <td>`0.2.0`</td>
+</tr>
+</tbody>
+</table>
```

Comment:
Is it possible that we can still keep the markdown syntax style, but using HTML labels in each table cell?

The reason is that using markdown table syntax is easy to read from pure markdown doc, and not error-prone (Because writing HTML in MD doesn't have syntax check, so it will easily miss some labels and cannot be detected by the editor).

Reply:
For example, like:

```md
| Markdown Column | HTML Column                     |
|-----------------|---------------------------------|
| *Italic*        | <span style="color:blue">Blue</span> |
```

Reply:
The reason why there is a wide table is because MD doesn't support line wrap for table cell, so if we want to add more explanation, then the line will be very long. If we can use HTML labe inside the table cell, then we can add the line break and other label, that will help the "wide" tables, also easy to review as a pure MD.

Reply:
With markdown tables, you have to keep each row as one line in the MD source. Adding HTML tags into the cell texts will further increase the width of each line. Am I misunderstanding anything?

Reply:
I see. Thanks for the explanation, I may have misunderstood something.